src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {

```

Comment:
I see few issues with this interface:
- The name of your interface is "AreaCalculator" whereas its method is named "getArea()" which is a void type. In Java naming convention, by 'get' prefix you imply that your method is supposed to return something and which you're not doing.
- Additionally, it doesn't actually provide information just on area but also other properties of the figure.

Please see the task requirement:
"behaviour - we can obtain the area of any figure and are able to draw it. To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override toString() method for this)."

How about 'Drawable' interface with 'draw()' method?